### PR TITLE
Check

### PR DESCRIPTION
**Pull request by seabbs**
Analysis is as written in the letter as far as  I can see. Data cleaning for the data that is downloaded looks correct and data cleaning for data that isn't looks sensible. 

Data missing from line 40 with no indicator in the README etc that the code isn't runnable or what needs to be present for it to be so. It would be nice if there was some flags to indicate this but it is stated in the letter by omission. 

Changes in this PR are all cosmetic and mostly include adding all dependencies to the library list at the top of the script. These may have been intentionally left out (`covidregionaldata` for example is not really needed for the meat of the analysis so it makes some sense not requiring someone just having a look to install which may have been the logic). I've also been a bit of a pedant about spaces and function arguments but obviously these are very optional tweaks.

### CODE REVIEW

File: R/lfd.r [hunk 1]
```diff
@@ -1,10 +1,17 @@
 library("readODS")
+library("readxl")
```
Comment (sbfnk) [inline R/lfd.r]: Only need because of a stray `read_excel` which has now been removed in `main` - this line can go now

File: R/lfd.r [hunk 2]
```diff
@@ -141,4 +147,4 @@ sums <- dfb %>%
   summarise_at(vars(positive, negative, total), sum)
 
 fp <- qbinom(c(0.025, 0.5, 0.975), sums$total, (1 - res$estimate))
-sums$positive - fp
+sums$positive - fp
```
Comment (sbfnk) [inline R/lfd.r]: Who ends a file with a line of code?

**Review by sbfnk — CHANGES_REQUESTED** [review]
Thanks!

File: R/lfd.r (hunk 2 above)
Comment (seabbs) [reply R/lfd.r]: Someone deeply traumatised by the potential for hidden white space

**Comment by seabbs** [comment]
Ah totally missed that the temporary file name was the same as that being read in for the first sheet. The wording in the letter made me think that only code or only some of the data was present.

**Review by sbfnk — APPROVED** [review]

**Pull request merged**